manifests/machineconfigdaemon/daemonset.yaml

```diff
@@ -79,6 +79,29 @@ spec:
           name: proxy-tls
         - mountPath: /etc/kube-rbac-proxy
           name: mcd-auth-proxy-config
+      - name: crio-kube-rbac-proxy
```

Comment:
is there a card linked to this? Interested to see what the reasoning for this is.